src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+	margin: 0px;
+}
+
+div {
+	height: 100px;
+	width: auto;
+}
+
+#red {
```

Comment:
do not style id, use only classes in CSS(ok to use tags for reset like body margin)